fusor/fusor.py

```diff
@@ -732,23 +738,23 @@ def add_gene_descriptor(self, fusion: Fusion) -> Fusion:
         if fusion.type == FusionType.CATEGORICAL_FUSION:
             properties.append(fusion.critical_functional_domains)
 
-        for property in properties:
-            for obj in property:
-                if "gene_descriptor" in obj.__fields__.keys():
+        for prop in properties:
+            for obj in prop:
+                if "gene_descriptor" in obj.model_fields.keys():
                     label = obj.gene_descriptor.label
                     norm_gene_descr, _ = self._normalized_gene_descriptor(
                         label, use_minimal_gene_descr=False
                     )
                     if norm_gene_descr:
                         obj.gene_descriptor = norm_gene_descr
         if fusion.regulatory_element and fusion.regulatory_element.associated_gene:
-            re = fusion.regulatory_element
-            label = re.associated_gene.label
+            reg_el = fusion.regulatory_element
```

Comment:
`re` is used for regex operations in python, so I renamed this. Especially because I added an import for it in this PR to check regex